fern/apis/generators-yml/definition/group.yml

```diff
@@ -63,9 +63,15 @@ types:
   GithubConfigurationSchema:
     discriminated: false
     union:
+      - GithubSelfhostedSchema
       - GithubCommitAndReleaseSchema
       - GithubPullRequestSchema
       - GithubPushSchema
+  
+  GithubSelfhostedSchema: 
```

Comment:
nit: I'd consider this `GithubUriSchema` since it's not necessarily just used in self-hosted environments. The name is less important here though - the property names are what matter.

Reply:
@amckinney in a bit of a rush, will follow up since a rename isnt a break for us